src/app/data-explorer/data-explorer.module.ts

```diff
@@ -77,6 +77,7 @@ import { MeqlOutputComponent } from './meql-output/meql-output.component';
     SearchFiltersComponent,
     CatalogueSearchFormComponent,
     QueryBuilderComponent,
+    MeqlPipe,
```

Comment:
Shouldn't need to import the `MeqlPipe`, the `MeqlOutputComponent` should be enough.

Reply:
Removed as not needed